crates/oxc_linter/src/rules/jsx_a11y/iframe_has_title.rs

```diff
@@ -0,0 +1,143 @@
+use oxc_ast::{
+    ast::{Expression, JSXAttributeValue, JSXElementName, JSXExpression, JSXExpressionContainer},
+    AstKind,
+};
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+use crate::{
+    context::LintContext,
+    rule::Rule,
+    utils::{get_prop_value, has_jsx_prop_lowercase},
+    AstNode,
+};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("eslint(iframe-has-title): Missing `title` attribute for the `iframe` element.")]
+#[diagnostic(severity(warning), help("Provide title property for iframe element."))]
+struct IframeHasTitleDiagnostic(#[label] pub Span);
+
+#[derive(Debug, Default, Clone)]
+pub struct IframeHasTitle;
+
+declare_oxc_lint!(
+    /// ### What it does
+    ///
+    /// Enforce iframe elements have a title attribute.
+    ///
+    /// ### Why is this bad?
+    ///
+    /// Screen readers alert users to the presence of a heading tag.
+    /// If the heading is empty or the text cannot be accessed,
+    /// this could either confuse users or even prevent them
+    /// from accessing information on the page's structure.
+    ///
+    /// ### Example
+    /// ```javascript
+    /// // Bad
+    /// <h1 />
+    ///
+    /// // Good
+    /// <h1>Foo</h1>
+    /// ```
```

Comment:
needs updating? 🙂

Reply:
Oh...Sorry for mistake 🙇 I updated in 6174e46 👍